### ISSUE TITLE
Allowed to add a data source for another Organization?

### ISSUE THREAD

**Issue by PRijnbeek** [EHDEN/ARACHNE#14]
When i go to my settings i can add my data sources but in the form i can also select another organization? should that not only be my own? or do i have some sort of admin rights?

![image](https://user-images.githubusercontent.com/7252105/118128276-f9424d80-b3fa-11eb-97ae-70d7227afc13.png)

**Comment by gklebanov** [comment]
Yes, this is a bug - you should only be choosing the one you created or the new one you are creating. 

@konstjar - please log
